src/test/java/htsjdk/samtools/util/SamLocusIteratorTest.java

```diff
@@ -287,6 +287,57 @@ public void testSimpleInsertion() {
         }
     }
 
+    /**
+     * Test insertion quality filter
+     */
+    @Test
+    public void testInsertionQualityFilter() {
+        final SAMRecordSetBuilder builder = getRecordBuilder();
+        // add records up to coverage for the test in that position
+        final int startPosition = 165;
+        String qualityString = "++++++++++,,,+++++++++++++++++***+++";
+        for (int i = 0; i < coverage; i++) {
+            // add a negative-strand fragment mapped on chrM with base quality of 10
+            builder.addFrag("record" + i, 0, startPosition, true, false, "10M3I17M3I3M", qualityString, 10);
+        }
+        final int insStart1 = 174;
+        final int insStart2 = 194;
+        // test both for include indels and do not include indels
+        for (final boolean incIndels : new boolean[] {false, true}) {
```

Comment:
If you have two different test cases it's usually better to make a data provider instead of iterating over them within the test.

Reply:
I think I'm going to just not include incIndels = false, because there is another test that tests this case and my change doesn't affect the behavior of the code when incIndels = false.